src/GlobalStyles.js

```diff
@@ -1,13 +1,26 @@
 import { createGlobalStyle } from 'styled-components'
 
 const GlobalStyles = createGlobalStyle`
+
+:root{
+  --color-white: #fff;
+  --color-light-mint: #E0ECDE;
+  --color-dark-blue: #205072;
+  --color-light-green: #7BE495;
+  --color-petrol: #329D9C;
+  --color-petrol-21: rgba(50, 157, 156, 0.21);
+  --color-petrol-13: rgba(50, 157, 156, 0.13);
+  --color-light-petrol: #68B2A0;
+  --color-red: #F75010;
```

Comment:
Instead of adding the value in the name (red), find something more generic like `primary`. 
Otherwise, it's confusing if you add another theme.